src/dune_patch/dune_patch.ml

```diff
@@ -90,9 +90,11 @@ let patches_of_string patch_string =
 
 let prog =
   lazy
-    (match Bin.which ~path:(Env_path.path Env.initial) "patch" with
+    (let path = Env_path.path Env.initial in
+     let bins = [ "patch"; "gpatch" ] in
```

Comment:
Actually, shouldn't you lookup gpatch first?

Reply:
Yes, `patch` is more common. But the set of people that have both utilities would almost certainly prefer `gpatch`. Otherwise, why did they install it explicitly?

Reply:
Fair enough. Done that.